dev-scripts/deploy_custom_rh-che.sh

```diff
@@ -306,7 +306,8 @@ CHE_CONFIG_YAML=$(yq ".\"data\".\"CHE_KEYCLOAK_REALM\" = \"NULL\" |
                       .\"data\".\"che.jdbc.username\" = \"$RH_CHE_JDBC_USERNAME\" |
                       .\"data\".\"che.jdbc.password\" = \"$RH_CHE_JDBC_PASSWORD\" |
                       .\"data\".\"che.jdbc.url\" = \"$RH_CHE_JDBC_URL\" |
-                      .\"data\".\"CHE_LOG_LEVEL\" = \"plaintext\" " ${RH_CHE_CONFIG})
+                      .\"data\".\"CHE_LOG_LEVEL\" = \"INFO\"|
```

Comment:
Do we really need this change?

Reply:
That's a fix of a buggy change in a previous PR, that helped me debugging this PR and will certainly be useful for others.

Because reading JSON-formatted logs while in DEBUG log level makes it quite hard to understand logs when trying to debug.